README for performance tests

kasiaMarek opened the pull request.

kasiaMarek committed:
README for performance tests

ohad commented on performance/runtests.py:
```diff
@@ -66,33 +75,48 @@ def buildTimes(name, iterations):
     plt.savefig(PATH_TO_CHARTS + test[NAME] + ".png")
     plt.clf()
 
+# List of all the tests to be runned, each ditectory has the following keys:
```
```suggestion
# List of all the tests to be run, each directory has the following keys:
```

ohad committed:
Update performance/runtests.py

Commit:
Tweak figure-generating scripts

Add a 'verbose' command line argument
Make Y-axis limit into a uniform parameter of the script
Tweak colours to be consistent with the manuscript

Commit:
Add parsec test to group tests.
